src/clients/node/docs.zig

```diff
@@ -88,7 +88,10 @@ pub const NodeDocs = Docs{
     ,
 
     .install_commands = "npm install tigerbeetle-node",
-    .build_commands = "npm install typescript @types/node && npx tsc --allowJs --noEmit main.js",
+    .build_commands =
+    \\npm install typescript @types/node
+    \\npx tsc --allowJs --noEmit main.js
```

Comment:
If you make a change to these docs.zig files, the build will fail until you:

```
zig/zig build client_docs -- --no-validate
```

as mentioned here https://github.com/tigerbeetle/tigerbeetle/tree/main/src/clients#documentation

Reply:
Thanks @eatonphil, you saved me some time figuring it out!

One problem we have is that the node client is running on Windows, but it can´t build on Windows yet because of the `sh` scripts used in `package.json`.
I'm just checking if it's easy enough make them cross-platform scripts, otherwise for now we can partially support running but not building on Windows.